modules/installing-ocp-agent-ibm-z.adoc

```diff
@@ -18,4 +18,82 @@ Depending on your {ibm-z-name} environment, you can choose from the following op
 [NOTE]
 ====
 Currently, ISO boot support on {ibm-z-name} (`s390x``) is available only for {op-system-base-full} KVM, which provides the flexibility to choose either PXE or ISO-based installation. For installations with z/VM and Logical Partition (LPAR), only PXE boot is supported.
+====
+
+[id="networking-reqs-ibmz_{context}"]
+==  Networking requirements for {ibm-z-title}
+
+In {ibm-z} environments, advanced networking technologies like Original Storage Architecture (OSA), HiperSockets, and Remote Direct Memory Access (RDMA) over Converged Ethernet (RoCE) require specific configurations that deviate from the standard settings used in the Agent Based Installer deployments. These overrides are necessary to accommodate their unique requirements and ensure a successful and efficient deployment on {ibm-z}.
```

Comment:
Per the [ISG word usage](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#l) entry for "like", it should not be used in place of "such as":
```suggestion
In {ibm-z} environments, advanced networking technologies such as Original Storage Architecture (OSA), HiperSockets, and Remote Direct Memory Access (RDMA) over Converged Ethernet (RoCE) require specific configurations that deviate from the standard settings used in the Agent Based Installer deployments. These overrides are necessary to accommodate their unique requirements and ensure a successful and efficient deployment on {ibm-z}.
```